polplot.py

```diff
@@ -5,10 +5,13 @@
 from matplotlib import rc
 from matplotlib.patches import Polygon
 from matplotlib.collections import PolyCollection, LineCollection
-#from pysymmetry.sunlight import terminator
 
+# Added by AØH 17/06/2022 for Lompe, should be changed when merged with Lompe 
+from lompe.utils.sunlight import terminator
```

Comment:
I think we should avoid having the polplot code being dependent on lompe... Or what do you think? The question then is what we can do to have the terminator plotting functionality included. One option is to copy the necessary functions into polplot. I could probably do that, and set it up to work if it is helpful?

Reply:
The original import for terminator calculations is pysymmetry.sunlight, though this only gives a dependency on a different package. I agree that the polplot code should be independent, so moving terminator calculations to the polplot code would work (the terminator function doesn't work as it is now anyways).

Reply:
I'm merging, but I will fix the dependency issue at some point. The terminator plotting functions are nice to have, so it's worth fixing.